File: cses/template/template.c
#include <stdio.h>
#include <stdlib.h>

int main(void) {
    int n;
    if (scanf("%d", &n) != 1) return 0;

    int *a = (int*)malloc(sizeof(int) * n);
    int *b = (int*)malloc(sizeof(int) * n);
    if (!a || !b) return 0;

    for (int i = 0; i < n; ++i) {
        scanf("%d %d", &a[i], &b[i]);
    }

    // ----- YOUR LOGIC HERE -----
    long long ans = 0;
    // ---------------------------

    printf("%lld\n", ans);

    free(a); free(b);
    return 0;
}

File: cses/template/Template.java
import java.io.*;
import java.util.*;

public class Template {
    static final class FastScanner {
        private final InputStream in;
        private final byte[] buffer = new byte[1 << 16];
        private int ptr = 0, len = 0;

        FastScanner(InputStream is) { in = is; }

        private int read() throws IOException {
            if (ptr >= len) {
                len = in.read(buffer);
                ptr = 0;
                if (len <= 0) return -1;
            }
            return buffer[ptr++];
        }

        int nextInt() throws IOException {
            int c, sgn = 1, x = 0;
            do c = read(); while (c <= ' ');         // skip spaces
            if (c == '-') { sgn = -1; c = read(); }
            while (c > ' ') { x = x * 10 + (c - '0'); c = read(); }
            return x * sgn;
        }
    }

    public static void main(String[] args) throws Exception {
        FastScanner fs = new FastScanner(System.in);
        PrintWriter out = new PrintWriter(new BufferedWriter(new OutputStreamWriter(System.out)));

        int n = fs.nextInt();
        int[] a = new int[n];
        int[] b = new int[n];
        for (int i = 0; i < n; i++) {
            a[i] = fs.nextInt();
            b[i] = fs.nextInt();
        }

        // ----- YOUR LOGIC HERE -----
        long ans = 0;
        // ---------------------------

        out.println(ans);
        out.flush();
    }
}
